gluten-ut/spark35/src/test/scala/org/apache/gluten/utils/clickhouse/ClickHouseTestSettings.scala

```diff
@@ -843,7 +843,6 @@ class ClickHouseTestSettings extends BackendTestSettings {
     .exclude("do not replace hash aggregate if child does not have sort order")
     .exclude("do not replace hash aggregate if there is no group-by column")
     .excludeGlutenTest("replace partial hash aggregate with sort aggregate")
-    .excludeGlutenTest("replace partial and final hash aggregate together with sort aggregate")
```

Comment:
why remove this? I don't see any change about ch backend.

Reply:
before, the CH backend exclude this UT, but now it is actually can enable due to https://github.com/apache/incubator-gluten/pull/7752/files#diff-8fd5d5c17d4e0d738c5089d703910bda7d1598728324a2987965d8c60fd5c9bbR89